nncf/quantization/advanced_parameters.py

```diff
@@ -128,6 +128,10 @@ class AdvancedQuantizationParameters:
     :type inplace_statistics: bool
     :param disable_bias_correction: Whether to disable the bias correction.
     :type disable_bias_correction: bool
+    :param alpha: SmoothQuant-related parameter. It regulates the calculation of the smooth scale.
+        The default value is 0.95. A negative value switches off the algorithm. In case of inaccurate results,
+        this parameter may be adjusted in the range from 0 to 1.
```

Comment:
```suggestion
    :param alpha: SmoothQuant-related parameter. It regulates the calculation of the smooth scale.
        The default value is 0.95. A negative value switches off the algorithm. In case of inaccurate results,
        this parameter may be adjusted in the range from 0 to 1 or set -1 to disable SmoothQuant algorithm.
```

Reply:
Thanks, updated